.vib/moodle/goss/moodle.yaml

```diff
@@ -0,0 +1,20 @@
+file:
+  /opt/bitnami/apache/conf/vhosts/moodle-vhost.conf:
+    exists: true
+    contains:
+    {{ range $rule := .Vars.apache.rules }}
+      - {{ $rule }}
+    {{ end }}
```

Comment:
In addition to my previous comment, I would recommend checking both moodle `moodle-vhost.conf` and `moodle-https-vhost.conf`.

Similar to https://github.com/bitnami/containers/blob/e4755eae4d659aa5206c0435e4511d79d9d136af/.vib/wordpress/goss/wordpress.yaml#L23-L37